frontend/src/javascript/utils/axiosUtils.js

```diff
@@ -1,5 +1,23 @@
 import axios from "axios";
 import toast, { getEndpointRelatedToast } from "@/javascript/utils/toastUtils";
+import { openModal } from "jenesius-vue-modal";
```

Comment:
Ehmm... domyślam się, że to były próby implementacji tego w modalu tak jak wcześniej mówiłem? Bo nie widzę, żeby to było jakoż używane. A nawet jakby było to nie jest najlepsze na takie rzeczy związane z vue. Do usunięcia ten dodany fragment.